app/backend/Scripts/pdf_to_image.py

```diff
@@ -1,13 +1,14 @@
-from transformer import transformer as tf
+from Transformer import Transformer as tf
 from PIL import Image
+import base64
```

Comment:
Reviewing from a non-Dev computer so I can't see dependencies but is base64 used here? If not we can remove it

Reply:
removed